dspace-api/src/main/java/org/dspace/access/status/DefaultAccessStatusHelper.java

```diff
@@ -91,157 +176,159 @@ public String getAccessStatusFromItem(Context context, Item item, LocalDate thre
+            // take the highest date (account for rare cases where more than one resource
+            // policy exists)
+            if (availabilityDate == null) {
+                availabilityDate = startDate;
+            } else if (takeMostRecentDate) {
+                availabilityDate = startDate.isBefore(availabilityDate) ? startDate : availabilityDate;
+            } else {
+                availabilityDate = startDate.isAfter(availabilityDate) ? startDate : availabilityDate;
+            }
         }
-
-        embargoDate = this.retrieveShortestEmbargo(context, bitstream);
-
-        return embargoDate != null ? embargoDate.toString() : null;
+        return availabilityDate;
     }
 
     /**
+     * Look at the DSpace object availability date to determine an access status value.
      *
+     * If the object is null, returns the "metadata.only" value.
+     * If there's no availability date, returns the "open.access" value.
+     * If the availability date is after or equal to the embargo
+     * threshold date, returns the "restricted" value.
+     * Every other cases return the "embargo" value.
+     *
+     * @param availabilityDate  the DSpace object availability date
+     * @param threshold         the embargo threshold date
+     * @return an access status value
      */
-    private LocalDate retrieveShortestEmbargo(Context context, Bitstream bitstream) throws SQLException {
-        LocalDate embargoDate = null;
-        // Only consider read policies.
-        List<ResourcePolicy> policies = authorizeService
-                .getPoliciesActionFilter(context, bitstream, Constants.READ);
-
-        // Looks at all read policies.
-        for (ResourcePolicy policy : policies) {
-            boolean isValid = resourcePolicyService.isDateValid(policy);
-            Group group = policy.getGroup();
-
-            if (group != null && StringUtils.equals(group.getName(), Group.ANONYMOUS)) {
-                // Only calculate the status for the anonymous group.
-                if (!isValid) {
-                    // If the policy is not valid there is an active embargo
-                    LocalDate startDate = policy.getStartDate();
-
-                    if (startDate != null && !startDate.isBefore(LocalDate.now())) {
-                        // There is an active embargo: aim to take the shortest embargo (account for rare cases where
-                        // more than one resource policy exists)
-                        if (embargoDate == null) {
-                            embargoDate = startDate;
-                        } else {
-                            embargoDate = startDate.isBefore(embargoDate) ? startDate : embargoDate;
-                        }
-                    }
-                }
-            }
+    private String getAccessStatusFromAvailabilityDate(LocalDate availabilityDate, LocalDate threshold) {
+        // If there is no availability date, it's an open access.
```

Comment:
What about restricted Bitstreams without dates? 
e.g. if you set an `administrator` ACO in submission or bulk access control it would be marked as open access even though it should be restricted.

Reply:
This will be restricted as there will be no policy applicable to the guest or current user.

![{47573409-7D34-49DF-B8F3-D0D85A191CD1}](https://github.com/user-attachments/assets/354f2672-0b48-4501-a081-ad90cbc06dfb)